src/backend/aspen/api/schemas/usergroup.py

```diff
@@ -8,11 +8,14 @@
 
 
 class GroupCreationRequest(BaseRequest):
-    name: constr(min_length=3, max_length=128, strict=True)  # type: ignore
-    prefix: constr(min_length=2, max_length=20, strict=True)  # type: ignore
-    address: Optional[constr(min_length=1, max_length=128, strict=True)]  # type: ignore
-    division: Optional[constr(min_length=1, max_length=128, strict=True)]  # type: ignore
-    location: Optional[constr(min_length=1, max_length=128, strict=True)]  # type: ignore
+    name: constr(min_length=3, max_length=1000, strict=True)  # type: ignore
```

Comment:
This has to be smaller, since we use `group.name` as part of an s3 object key for sequence downloads. s3 object key names have a limit of 1024-bytes UTF-8 encoded, which is roughly the same number of characters since we mostly use ASCII chars. So an actual 1000 char name will break that endpoint. I think 512 or even 256 is much more reasonable here.

Reply:
See https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-keys.html

Reply:
Ah, thanks for pointing that out. I didn't realize it got used in S3 keys.